### PR TITLE
Feat/414068  show breach reasons post

### PR DESCRIPTION
**Pull request by whitewaterdesign**

### CODE REVIEW

File: app/models/mappers/check-activities.js [hunk 1]
```diff
@@ -322,13 +322,30 @@ const getActivityLabelFromCreatedDog = (createdDogEvent) => {
...
+/**
+ * @param {[string, string][]} breaches
+ * @return {string[]}
+ */
+const mapBreachesToArray = (breaches) => {
+  return breaches.reduce((filteredBreaches, breach) => {
+    const [key, label] = breach
+    const dogBreachRgx = /dog_breaches\/[0-9]+\[]/
+    if (dogBreachRgx.test(key)) {
+      return [...filteredBreaches, [label]]
```
Comment (jbarnsley10) [inline app/models/mappers/check-activities.js:334]: SonarCloud doesn't like this

File: app/models/mappers/check-activities.js [hunk 2]
```diff
@@ -322,13 +322,30 @@ const getActivityLabelFromCreatedDog = (createdDogEvent) => {
...
 }
 
+/**
+ * @param {[string, string][]} breaches
+ * @return {string[]}
+ */
+const mapBreachesToArray = (breaches) => {
+  return breaches.reduce((filteredBreaches, breach) => {
+    const [key, label] = breach
+    const dogBreachRgx = /dog_breaches\/[0-9]+\[]/
```
Comment (jbarnsley10) [inline app/models/mappers/check-activities.js:332]: SonarCloud doesn't like this

File: app/models/mappers/check-activities.js [hunk 3]
```diff
@@ -300,9 +301,8 @@ const activityLabels = {
  * @param {string} eventType
  * @returns {GetActivityLabelFromAuditFieldRecordFn}
  */
-const getActivityLabelFromAuditFieldRecord = (eventType) => (auditFieldRecord) => {
+const getActivityLabelFromAuditFieldRecord = (eventType) => (auditFieldRecord, children) => {
```
Comment (jbarnsley10) [inline app/models/mappers/check-activities.js:304]: Don;t think the children param is used

**Review by jbarnsley10 — CHANGES_REQUESTED** [review]
More great work. Only a minor change or two

File: app/models/mappers/check-activities.js (hunk 3 above)
Comment (whitewaterdesign) [reply app/models/mappers/check-activities.js]: good spot

**Review by jbarnsley10 — APPROVED** [review]
Great

**Pull request merged**